CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# 0.15.0
+
+## Features
+
+- Add support for repository names with sub-namespaces - this can be achieved
+  with GitLab.
```

Comment:
I am not sure about the word `achieved` -- Maybe `it is possible`?

Note that also Pagure can have subgroups or empty groups if I am right. That did not work as well.

Reply:
oki, will update

Reply:
thanks